apps/api/.env.example

```
@@ -0,0 +1,14 @@
+# Default environment variables for Paramarsh SMS API
+
+# Server Configuration
+PORT=3005
+
+# Database Configuration
+DATABASE_URL="file:./dev.db"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)